pkg/pipelines/bootstrap.go

```diff
@@ -43,14 +53,14 @@ func Bootstrap(quayUsername, baseRepo, prefix string) error {
 	}
 	outputs = append(outputs, githubAuth)
 
-	authJSONPath, err := pathToDownloadedFile(quayUsername + "-auth.json")
```

Comment:
Quay recommends robot accounts over personal accounts.   The default file name for robot auth file is <user>-robot-auth.json.   See https://github.com/rhd-gitops-example/deployment-pipelines/blob/master/Example.md